src/components/ButtonPanel.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Button from './Button';
+
+export default function ButtonPanel() {
+  const buttons = [['AC', '+/-', '%', '÷'],
+    ['7', '8', '9', 'X'],
+    ['4', '5', '6', '-'],
+    ['1', '2', '3', '+'],
+    ['0', '.', '='],
+  ];
+
+  return (
+    <div>
+      {buttons.map(e => (
+        <div key={(Math.random() * 100).toString()}>
```

Comment:
using a random value a the key for a component is counter productive to the idea of keys, please see this [link](https://stackoverflow.com/questions/29808636/when-giving-unique-keys-to-components-is-it-okay-to-use-math-random-for-gener)